src/report/user-report.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { Length, IsString, IsEnum } from 'class-validator';
+import { UserReportStatus } from '.';
+
+export class ReportDto {
+    @IsString()
+    @Length(5, 256)
+    reason: string;
+}
+export class UpdateStatusDto {
```

Comment:
1. this file should be in `dto/` directory
2. every dto should be in separate file, they are small so that seems like an unnecessary job but trust me (`src/report/dto/create-report.dto.ts`, `src/report/dto/update-status.dto.ts`)
3. rename ReportDto to CreateReportDto

minor naming convention issues, but still nice job
someone s gotta complain about something, right? ;)